components/lib/datatable/DataTable.js

```diff
@@ -105,6 +105,25 @@ export const DataTable = React.forwardRef((inProps, ref) => {
         }
     });
 
+    const [bindDocumentTouchMoveListener, unbindDocumentTouchMoveListener] = useEventListener({
+        type: 'touchmove',
+        listener: (event) => {
+            if (columnResizing.current) {
+                onColumnResize(event);
+            }
+        }
+    });
+
+    const [bindDocumentTouchEndListener, unbindDocumentTouchEndListener] = useEventListener({
+        type: 'touchend',
+        listener: () => {
+            if (columnResizing.current) {
```

Comment:
These conditions are the same as when binding mouse events, should they be refactored?

Reply:
yes